dev-docs/analytics/adnuntius.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: analytics
+title: Livewrapped
+description: Livewrapped Analytics Adapter
+modulecode: livewrapped
+prebid_member: true
```

Comment:
```suggestion
prebid_member: true
enable_download: false
tcfeu_supported: true
usp_supported: false
gvl_id: 919
```

adding missing meta data. Assuming that `919` is also supported like in the bidder.